packages/swirl-components/src/components/swirl-image-grid-item/swirl-image-grid-item.tsx

```diff
@@ -30,7 +31,7 @@ export class SwirlImageGridItem {
   @Prop() gifPlayLabel?: string = "Continue GIF playback";
   @Prop() icon?: string;
   @Prop() interactive?: boolean;
-  @Prop() loading?: SwirlImageGridItemLoading;
+  @Prop() loading?: SwirlImageGridItemLoading = "intersecting";
```

Comment:
Question: Technically a breaking change. Is this safe? I'm not sure where this component is used by now.

Reply:
probably not. removed it just to be safe